app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPicker.kt

```diff
@@ -34,32 +34,30 @@ import com.wire.android.ui.common.imagepreview.BitmapState
 import com.wire.android.ui.common.imagepreview.BulletHoleImagePreview
 import com.wire.android.ui.common.textfield.WirePrimaryButton
 import com.wire.android.ui.theme.wireTypography
+import com.wire.android.ui.userprofile.UserProfileViewModel
 
-
-@OptIn(ExperimentalMaterialApi::class)
+@OptIn(ExperimentalMaterialApi::class, ExperimentalMaterial3Api::class)
 @Composable
 //TODO: the input data for ProfileImageScreen will be decided later on after sync with Yamil
-fun ImagePicker(
-    initialBitmap: Bitmap,
-    onCloseClick: () -> Unit,
-    onConfirmPick: (Bitmap) -> Unit
+fun AvatarPickerScreen(
+    viewModel: UserProfileViewModel
```

Comment:
I though we wanted to make this component as stand-alone component but now it has dependency on UserProfileviewModel hmmmm it has now also access to all the methods that should not really have access like logout, editProfile etc....

Reply:
Let's just keep it as it is for now as it falls out of the scope of this PR. But will definitely need to refactor it to have our own `AvatarPickerViewModel` injected instead, handling the logic passing the chosen image uri.